geometry_msgs/Pose[] poses
int32[] ids
string[] colors
---
